test/unistd/fstatfs.c

```diff
@@ -8,14 +8,35 @@ int main() {
   struct statfs buf;
 
   int rtn;
-  assert(fstatfs(STDOUT_FILENO, &buf) == 0);
+  int fstatfs_rtn = fstatfs(STDOUT_FILENO, &buf);
   printf("f_type: %ld\n", buf.f_type);
+  assert(fstatfs_rtn == 0);
+  printf("f_blocks: %d\n", buf.f_blocks);
+  assert(buf.f_blocks == 1000000);
 
   int f = open("file", O_RDWR | O_CREAT);
-  assert(fstatfs(f, &buf) == 0);
+  fstatfs_rtn = fstatfs(f, &buf);
   printf("f_type: %ld\n", buf.f_type);
+  printf("f_blocks: %d\n", buf.f_blocks);
+#if NODEFS
```

Comment:
Should this be `NODEFS || NODERAWFS`? Or are they expected to behave differently?

Reply:
Yes, NODEFS and NODERAWFS should work the same. [b1b9b01](https://github.com/emscripten-core/emscripten/pull/23912/commits/b1b9b01bec4510ca593b880396b0755febeff547)